addons/mrp/migrations/13.0.2.0/post-migration.py

```diff
@@ -152,7 +152,7 @@ def fill_unbuild_company_id(cr):
         SET company_id = mb.company_id
         FROM mrp_bom mb
         WHERE mu.bom_id = mb.id AND mb.company_id IS NOT NULL
-            AND mu.company_id IS NULL"""
+            AND mu.company_id != mb.company_id"""

```

Comment:
Better to use this operator for avoiding problems with NULL values:

```suggestion
            AND mu.company_id IS DISTINCT FROM mb.company_id"""

```